telemetry/specs/INT.mdk

```diff
@@ -711,6 +711,26 @@ INT shim header for VXLAN GPE encapsulation:
 * Type: This field indicates the type of INT Header following the shim header.
 Two Type values are used: one for the hop-by-hop header type and the other for
 the destination header type (See Section [#sec-int-header-types]).
+* G: Indicates whether the original packet (before insertion of INT headers
+and metadata) used a VXLAN or VXLAN GPE encapsulation.
+  - 0: Original packet used VXLAN encapsulation.
+  - 1: Original packet used VXLAN GPE encapsulation.
+  - This may be used as a hint that helps the INT sink (when it is not the VTEP)
+    determine whether to progress the packet using a VXLAN GPE encapsulation,
+    or whether to convert the VXLAN GPE encapsulation back to a VXLAN (without
+    GPE) encapsulation.
+  - Note that this bit is only set by v1.1 implementations. In mixed
+    environments with both v1.0 and v1.1 nodes, the INT sink should ignore the
+    value of this bit and rely on local configuration to determine whether to
+    convert the VXLAN GPE encapsulation back to a VXLAN encapsulation.
+    Typically in such a mixed or v1.0 environment, INT would only be inserted
+    into one type of encapsulation but not the other, e.g. VXLAN but not
+    VXLAN GPE since the INT sink always converts VXLAN GPE packets with INT
+    back to the VXLAN encapsulation.
+  - In an all v1.1 environment, both packets received with VXLAN encapsulation
+    and VXLAN GPE encapsulation could have INT inserted, with the INT sink
+    relying on the value of this bit to determine whether to carry out the
+    conversion from VXLAN GPE back to VXLAN.
```

Comment:
Since this is now aimed at the INT v2.0 spec, the two bullet points about v1.0, v1.1 and interop can just be removed.

Reply:
+1

Reply:
Done